src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,208 @@ set of steps, guiding toward the completion of a specified process.
 
 ## Overview
 
-Use progress indicators to keep the user on track when completing a specific
-task. By dividing the end goal into smaller, sub-tasks, it increases the
-percentage of completeness as each task is completed.
+Progress indicators help manage a user's expectations when completing a
+multistep process. They show which step the user is currently on, the total
+number of steps, and overall progress in completing a task.
 
-## Best practices
+<Row>
+<Column colLg={8}>
+
+![Example of progress indicator used in a payment UI](images/progress-indicator-usage-1.png)
+
+</Column>
+</Row>
+
+### When to use
+
+- When the user is working through a linear process that can be organized into
+  three or more steps.
+- When the user could benefit from understanding their progress on long forms
+  such as eCommerce checkouts, onboarding, or visa applications.
+- When user inputs should be validated before progressing to the next step.
+- To complement standard back/next navigation in a linear sequence.
+
+### When not to use
 
-#### Logical progression
+- When a process or form has fewer than three steps.
+- When the process may be completed in any order.
+- When the number of steps may change based on conditional logic.
 
-Display the steps in order from left to right. Indicate to the user that they
-are performing a multistep process, and show the direction of movement. Allow
-the user to return to a previous step to review their data submission.
+### Variants
 
-#### Indicate the current step
+| Variant     | Purpose                                                            |
+| ----------- | ------------------------------------------------------------------ |
+| Default     | Allows users to see where they are in a linear, multistep process. |
+| Interactive | Allows users to return to a previously completed step.             |
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Progress indicator anatomy](images/progress-indicator-usage-2.png)
+
+</Column>
+</Row>
 
-Keeping the user informed of where they currently are within the process or task
-at hand will give them a sense of control. This helps the user to know where
-they are in relation to where they have been, and what sections are to follow.
-Clear labels should accompany the progress indicator to indicate what the user
-will accomplish within each step. Keep labels between one to two words.
+1. **Status indicator:** Communicates if a step is completed, incomplete, not
+   started, or has an error. See the
+   [status indicator pattern](/patterns/status-indicator-pattern/) for guidance.
+2. **Active step line:** Indicates completed steps and the step the user is
+   currently on.
+3. **Label:** Communicates what the user will accomplish in each step. Numbering
```

Comment:
Also check in with @mbgower Should we call this Label and the other is Helper text? Rather than Optional label?

Reply:
I'll have Gower do a full review.